conf.py

```diff
@@ -347,6 +347,7 @@ def __init__(self, **options):
     'seaborn': ('https://seaborn.pydata.org/', None),
     'skimage': ('https://scikit-image.org/docs/stable/', None),
     'statsmodels': ('https://www.statsmodels.org/stable/', None),
+    'imagio':('https://imageio.readthedocs.io/en/stable/', None),
```

Comment:
The `imageio` link is not rendering. Does this look right? Is 
```
:mod:`imageio`
```
correct below?